src/webui/src/components/TrialsDetail.tsx

```diff
@@ -142,10 +144,11 @@ class TrialsDetail extends React.Component<{}, TrialDetailState> {
                             </Pivot>
                         </div>
                         {/* trial table list */}
-                        <div style={{ backgroundColor: '#fff' }}>
-                            <Stack horizontal className='panelTitle' style={{ marginTop: 10 }}>
-                                <span style={{ marginRight: 12 }}>{tableListIcon}</span>
-                                <span>Trial jobs</span>
+                        <div style={{ backgroundColor: '#fff', marginTop: 18 }}>
```

Comment:
suggest moving the style into `trialsDetail.scss`

Reply:
Ok. will to do.

Reply:
add `bulletedList` className for background color style. and keep style={{ marginTop:18 }} because of priority(write this style in .bulletedList doesn't work)